cachi2/core/package_managers/pip.py

```diff
@@ -25,7 +25,6 @@
     Pattern,
     Union,
     cast,
-    no_type_check,
```

Comment:
**Commit message:**

> Resolves #485

Please use direct URLs instead of numbers in the commit messages so that it's properly rendered in local git histories.

Also, thanks to your example I think I now know what I recently did wrong in one of my commit messages where I used the keyword, but GitHub didn't close the issue and I had to close it manually - you don't use a colon! You may be onto something :) .